system/system_monitor/include/system_monitor/utils.hpp

```diff
@@ -19,7 +19,7 @@
 
 /// This function will spin the given node and update the diagnostic state at each iteration
 /// \tparam MonitorT Monitor type
-/// \param monitor_ptr Shared pointer of a monitor node to be spinned.
+/// \param monitor_ptr Shared pointer of a monitor node to be spin().
 /// \param period Spin period.
```

Comment:
How about moving this right before the function (around Line32)?

Reply:
Also, using `spin() in already spinning node` is not allowed in ROS2, so we should confirm whether we can still use this function or not.

Reply:
Moved comments in 3213d564af40f917e23acc3ab6108b66c4afe77d (and slightly formatted).
Deleting the function might be out-of-scope of this PR. Better to do on another PR?

Reply:
@kmiya Thank you!
> Deleting the function might be out-of-scope of this PR. Better to do on another PR?

Yes, it's fine.